examples/app_commands/slash_options.py

```diff
@@ -9,31 +9,29 @@
 @bot.slash_command(guild_ids=[...])
 async def hello(
     ctx: discord.ApplicationContext,
-    name: Option(str, "Enter your name"),
-    gender: Option(str, "Choose your gender", choices=["Male", "Female", "Other"]),
-    age: Option(int, "Enter your age", min_value=1, max_value=99, default=18)
+    name: str = Option(description="Enter your name"),
+    gender: str = Option(description="Choose your gender", choices=["Male", "Female", "Other"]),
+    age: int = Option(description="Enter your age", min_value=1, max_value=99, default=18)
     # passing the default value makes an argument optional
     # you also can create optional argument using:
-    # age: Option(int, "Enter your age") = 18
+    # age: int = Option("Enter your age", default=18)
 ):
     await ctx.respond(f"Hello {name}! Your gender is {gender} and you are {age} years old.")
 
 
 @bot.slash_command(guild_ids=[...])
 async def channel(
     ctx: discord.ApplicationContext,
-    channel: Option([discord.TextChannel, discord.VoiceChannel], "Select a channel")
-    # you can specify allowed channel types by passing a list of them like: [discord.TextChannel, discord.VoiceChannel]
+    channel: discord.enums.SlashCommandOptionType.channel = Option(description="Select a channel"),

```

Comment:
```suggestion
    channel: discord.SlashCommandOptionType.channel = Option(description="Select a channel"),
```

No need to go through enums to access this since it's imported via wildcard in `__init__.py`